logproof/src/linear_relation.rs

```diff
@@ -1197,7 +1198,7 @@ impl LogProof {
         // Make sure we have an equal number of values and bounds to serialize
         assert_eq!(values.len(), log_b.len());
 
-        let mut bitvec = BitVec::with_capacity(values.len() * log_b.iter().sum::<u64>() as usize);
+        let mut bitvec = BitVec::with_capacity(log_b.iter().sum::<u64>() as usize);
```

Comment:
Pretty sure this is correct, but double check me. We zip these two things together, and the `ith` elem in `values` ends up being `log_b[i]` long. Multiplying by the length was probably blowing this allocation up quite a bit.

Reply:
Ya makes sense.